cocos/2d/assembler/label/bmfontUtils.ts

```diff
@@ -161,6 +161,13 @@ export const bmfontUtils = {
         _spriteFrame = fontAsset.spriteFrame;
         _fntConfig = fontAsset.fntConfig;
         shareLabelInfo.fontAtlas = fontAsset.fontDefDictionary;
+        if (!shareLabelInfo.fontAtlas) {
+            if (comp.cacheMode === CacheMode.CHAR) {
+                shareLabelInfo.fontAtlas = new LetterAtlas(1024, 1024);
```

Comment:
It feels strange, if fontAtlas is missing and if it's an unexpected scenario, then maybe we should just skip rendering.
We should be always vigilant about memory creation, especially on GPU side, please justify the modification with fine cause

Reply:
![image](https://user-images.githubusercontent.com/32831993/180340301-82ca23fb-f8cb-42c1-a31c-34263c13d99c.png)
There are many usage of fontAtlas, we should create a default fontAtlas unless we make null judgment protection in each of the above places

Reply:
I create two global variables for assignment. It can avoid memory allocations.

Reply:
Ok, 64*64 seems much smaller and acceptable